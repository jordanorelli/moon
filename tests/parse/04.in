an_object: {
    item_one: 1
    item_two: "two"
    item_three: [1 2 3]
    item_four: {
        item_one: "one"
        item_two: 2
        item_three: ["one" "two" "three"]
    }
}
