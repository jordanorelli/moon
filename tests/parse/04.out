root:
  assign:
    name:
      root_int
    value:
      int:
        1
  assign:
    name:
      root_float
    value:
      float:
        1.2
  assign:
    name:
      root_exp
    value:
      float:
        1e+09
  assign:
    name:
      root_hex
    value:
      int:
        703710
  assign:
    name:
      root_oct
    value:
      int:
        511
  assign:
    name:
      an_object
    value:
      object:
        item_four:
          object:
            item_one:
              string:
                one
            item_three:
              list:
                string:
                  one
                string:
                  two
                string:
                  three
            item_two:
              int:
                2
        item_one:
          int:
            1
        item_three:
          list:
            int:
              1
            int:
              2
            int:
              3
        item_two:
          string:
            two
